package.json

```diff
@@ -20,23 +28,23 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
-    "accord": "0.16.x",
-    "autoprefixer-stylus": "0.0.2",
-    "axis": "0.3.x",
-    "clean-css": "3.x",
-    "coffee-script": "1.9.x",
-    "colors": "1.x",
-    "mocha": "2.x",
-    "monocle": "1.1.x",
-    "should": "5.x",
-    "stylus": "0.50.x",
-    "uglify-js": "2.x",
-    "umd": "3.x",
-    "when": "3.x",
-    "zombie": "3.x"
+    "autoprefixer-stylus": "latest",
+    "del": "latest",
+    "gulp": "latest",
+    "gulp-accord": "latest",
+    "gulp-babel": "latest",
+    "gulp-minify-css": "latest",
+    "gulp-rename": "latest",
+    "gulp-uglify": "latest",
+    "gulp-wrap-umd": "latest",
+    "mocha": "latest",
+    "should": "latest",
+    "stylus": "latest",
+    "when": "latest",
+    "zombie": "latest"
```

Comment:
I think that latest for devDeps is okay because it's only tests, which don't break the actual lib. I should change `mocha` back though because it's our testing suit. Otherwise the others should stay up to date on the latest. Moment.js does it this way as well